File: c++/khai_bao_lop_nhan_vien.cpp
#include <bits/stdc++.h>
using namespace std;
/*
Nguyen Van Hoa
Nam
11/22/1982
Mo Lao-Ha Dong-Ha Noi
8333123456
31/12/2013
*/

class NhanVien{
	private:
		string id, ten, sex, ns, dc, mst, hd;
	public:
		friend istream& operator >> (istream &in, NhanVien &a);
		friend ostream& operator >> (ostream &out, NhanVien a);	
};

istream& operator >> (istream &in, NhanVien &a){
	a.id = "00001";
	getline(in, a.ten);
	in >> a.sex >> a.ns;
	in.ignore();
	getline(in, a.dc);
	in >> a.mst >> a.hd;
	return in;
}

ostream& operator >> (ostream &out, NhanVien a){
	out << a.id << " " << a.ten << " "  << a.sex << " " << a.ns << " " << a.dc << " " << a.mst << " " << a.hd;
	return out;
}

int main(){
    NhanVien a;
    cin >> a;
    cout >> a;
    return 0;
}

File: java/javaPogg/src/khaiBaoLopNhanVien.java
import java.util.*;

class Employee{
    private String id, name, sex, ns, dc, mst, date;
    Employee(){}
    Employee(String id, String name, String sex, String ns, String dc, String mst, String date){
        this.id = id;
        this.name = name;
        this.sex = sex;
        this.ns = ns;
        this.dc = dc;
        this.mst = mst;
        this.date = date;
    }
    public String toString(){
        return id + " " + name + " " + sex + " " + ns + " " + dc + " " + mst + " " + date;
    }
}
        
public class khaiBaoLopNhanVien {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Employee nv = new Employee("00001", sc.nextLine(), sc.nextLine(), sc.nextLine(), sc.nextLine(), sc.nextLine(), sc.nextLine());
        System.out.println(nv);
    }
}
/*
Nguyen Van Hoa
Nam
22/11/1982
Mo Lao-Ha Dong-Ha Noi
8333123456
31/12/2013
*/